tests/dragonfly/connection_test.py

```diff
@@ -534,9 +534,13 @@ async def test_pipeline_batching_while_migrating(
     writer.write("INCR a\r\n".encode())
     await writer.drain()
 
+    async def read():
```

Comment:
care to add a comment to explain? :)

Reply:
Added a comment, it's only a suspicion so far 😅